core/dbt/events/functions.py

```diff
@@ -55,19 +55,10 @@
 # then we should override the logging stream to use the colorama
 # converter. If the TERM var is set (as with Git Bash), then it's safe
 # to send escape characters and no log handler injection is needed.
-colorama_stdout = sys.stdout
-colorama_wrap = True
-
-colorama.init(wrap=colorama_wrap)
-
 if sys.platform == "win32" and not os.getenv("TERM"):
-    colorama_wrap = False
-    colorama_stdout = colorama.AnsiToWin32(sys.stdout).stream
```

Comment:
Unless this statement has a side-effect or is being used externally, it appears to be deadcode, hence I removed it. Doing a grep over the `dbt-core` repo, I see no references outside of this file for `colorama_stdout`.

Reply:
@nathaniel-may Seems like you added this section